usecases/cluster/state.go

```diff
@@ -35,6 +35,7 @@ type Config struct {
 	IgnoreStartupSchemaSync bool       `json:"ignoreStartupSchemaSync" yaml:"ignoreStartupSchemaSync"`
 	SkipSchemaSyncRepair    bool       `json:"skipSchemaSyncRepair" yaml:"skipSchemaSyncRepair"`
 	AuthConfig              AuthConfig `json:"auth" yaml:"auth"`
+	AdvertiseAddr			string     `json:"AdvertiseAddr" yaml:"AdvertiseAddr"`
```

Comment:
I think that you need to fix the indentation here. 
In json and yaml annotations please use lower cased variable names `json:"advertiseAddr"`

There's also `AdvertisePort` I think that it would be good also to expose this setting